sopel_waifu/waifu.json

```
@@ -1885,6 +1750,180 @@
         "Isurugi Noe",
         "Andou Aiko"
     ],
+    "Type Moon": [
```

Probably should include the hyphen and format as [the "stylized" studio name](https://commons.wikimedia.org/wiki/File:Type-moon.svg):

```suggestion
    "TYPE-MOON": [
```